src/pages/search/index.tsx

```diff
@@ -287,18 +288,27 @@ const Search = () => {
                   <div className="text-sm my-4 md:mb-4 md:mt-0 lg:my-0" data-cy="number-of-results">
                     {status === "success" && renderNoOfResults()}
                   </div>
-                  <span className="text-sm mt-4 md:mt-0 text-right">
+                  <span className="text-sm mt-4 md:mt-0 text-right flex flex-wrap gap-x-2 md:justify-end">
+                    <span>Download data (.csv): </span>
                     <a
                       href="#"
                       className="text-blue-600 hover:underline flex gap-2 items-center justify-end"
                       data-cy="download-search-csv"
                       onClick={handleDownloadCsvClick}
                     >
-                      {downloadCSVStatus === "loading" && <Loading />} Download search results (.csv)
+                      {downloadCSVStatus === "loading" && <Loading />} this search
                     </a>
+                    <span>|</span>
+                    <ExternalLink
```

Comment:
Won't this also show while loading?

Reply:
Yes that's OK as both options are always available to click on, when loading the loading indicator is displayed next to the link that says "this search"

Reply:
okies -sgtm